compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -252,7 +252,9 @@ object Completion {
       (
```

Comment:
I would move the two added checks above this line. also update the comment at the top of this method.